tool/prdeployer.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:firebase/firebase_io.dart';
+import 'dart:async';
+import 'package:github/server.dart';
+import 'dart:io';
+
+GitHub github;
+String firebaseAuth;
+PullRequest currentPullrequest;
+
+String firebaseRoot = "https://flutter-web-controller.firebaseio.com/flutter/pulls";
+
+main(List<String> arguments) async {
+  String branchName = arguments[0];
+  Map env = Platform.environment;
+  github = createGitHubClient(auth: new Authentication.withToken(env["GITHUB_TOKEN"]));
+  firebaseAuth = arguments[1];
+
+  FirebaseClient fbClient = new FirebaseClient(firebaseAuth);
+
+  List prsList = await getPullRequests();
+  for (PullRequest p in prsList) {
+    if (branchName == p.head.ref) {
+      currentPullrequest = p;
+      break;
+    }
+  }
```

Comment:
instead of for loop: `currentPullRequest = prsList.firstWhere((p) => p.head.ref == branchName);`